### ISSUE TITLE
Account step: Buttons are missing hover style

### ISSUE THREAD

**Issue by nuriapenya** [Automattic/wp-calypso#100217]
## Problem
The account step on the onboarding presents four buttons to choose how to create an account, and they don't have any hover style, making it difficult for the users to identify what is selected. The same goes for keyboard tab navigation. This leads to accessibility issues.

## Where
On incognito, click "Get Started" from WordPress.com landing page. 

![Image](https://github.com/user-attachments/assets/9fcf5673-a1da-4e2f-b134-ed5eea085119)

## Solution
Add a hover effect like other buttons.
